app/src/renderer/vuex/modules/pool.js

```diff
@@ -7,7 +7,7 @@ export default ({ node }) => {
     loaded: false,
     error: null
   }
-  const state = JSON.parse(JSON.stringify(emptyState))
+  const state = emptyState
```

Comment:
if I edit now `state.pool.x` I change `emptyState.pool` as well. Cloning the objects is a safety pattern.